sm4_keyschedule.c

```diff
@@ -0,0 +1,180 @@
+/* ===================================================================== */
+/* This file is a little helper to compute SM4 key scheduling            */
+/* from any 4 consecutive round keys                                     */
+/* Original author: Sylvain Pelissier <sylvain.pelissier@gmail.com> 2022 */
+/*                                                                       */
+/* Usage:                                                                */
+/* sm4_keyschedule SM4_key_in_hex                                        */
+/* sm4_keyschedule Round_key_in_hex Round_key_number_between_0_and_32    */
+/*                                                                       */
+/* Examples:                                                             */
+/* sm4_keyschedule 11223344556677881122334455667788                      */
```

Comment:
These examples should be updated with 4/5 args, no?

Reply:
Yes I forgot them. Thank you.